Lombiq.NodeJs.Extensions/scripts/clean-assets.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @summary A script to clean asset files; expects configuration in assets-to-copy.json or package.json (assetsToCopy).
+ * @description This script removes assets previously copied using copy-assets.js.
+ */
+const path = require('path');
+const util = require('util');
+const rimraf = util.promisify(require('rimraf'));
+const getAssetsConfig = require('./get-assets-config');
+
+const verbose = true;
+const filePattern = '**/*';
+
+// Change to consuming project's directory.
+process.chdir('..');
+process.chdir('..');
```

Comment:
Is this meant to be twice? If so, use `process.chdir('../..')` instead. This works on Windows to if you were worried about that. You can also use `path.resolve('..', '..')` instead of `'../..'`.
Same [here](https://github.com/Lombiq/NodeJs-Extensions/blob/44782f9ed55eecba85faa109ea88b75f68d0bda7/Lombiq.NodeJs.Extensions/scripts/copy-assets.js#L17-L18).

Reply:
I felt like avoiding forward slashes in paths, if we're already using Node. And I didn't want to include `path` just for that. So I thought that the solution I came up with was both clear, platform-independent, and without needing another dependency.

Is it ugly to you?

Reply:
Not just ugly, but more importantly if I see two identical lines one after the other I get suspicious that it's maybe an accidental duplication. Worse yet, someone might act on that suspicion.

Also why are you against using forward slashes in Node paths?  Like I said, works perfectly fine in Windows.
![image](https://user-images.githubusercontent.com/4007293/182872269-36f155fe-d813-421e-b9e8-13533693003e.png)

Also also, there is no cost associated with including `path` because as a system dependency. It's loaded in by default:
![image](https://user-images.githubusercontent.com/4007293/182872500-1bee03b3-0005-4466-aa38-1742a998192e.png)
See? It was already there without needing a `require`.

Reply:
Convinced 😅 Thanks again for the insights.